mongo_connector/oplog_manager.py

```diff
@@ -459,6 +505,19 @@ def get_oplog_cursor(self, timestamp=None):
         If no timestamp is specified, returns a cursor to the entire oplog.
         """
         query = {'op': {'$ne': 'n'}}
+
+        if self.oplog_ns_set and not self.oplog_ex_ns_set:
+            query['ns'] = {'$in': self.oplog_ns_set}
+
+        if self.oplog_ex_ns_set and not self.oplog_ns_set:
+            query['ns'] = {'$nin': self.oplog_ex_ns_set}
+
+        if self.oplog_ns_set and self.oplog_ex_ns_set:
+            query = {'$and': [{'ns': {'$nin': self.oplog_ex_ns_set}},
+                              {'ns': {'$in': self.oplog_ns_set}}]}
+
+        LOG.debug("Cursor query: %s" % query)
```

Comment:
The cursor returned from get_oplog_cursor no longer filters by namespace. The filtering is instead done in the run method here: https://github.com/mongodb-labs/mongo-connector/pull/495/files#diff-9e5fc676a1f3b77e19871877266b4cf0R314

If you remove these lines and add the equivalent filtering logic using `self.oplog_ns_set` and `self.oplog_ex_ns_set` up there the tests should pass again.